coralproject/talk
# Action bugfixes

## What does this PR do?

Fixes the following bugs:

1. Reply button on replies does not open reply box.
2. Like button fails to work when replies are rendered on the screen.
3. Clicking "flag" and reloading the page increments the like count.
4. Notifications block the last comment in the stream.

## How do I test this PR?

1. Click the "Reply" button on a reply. The reply box should open on the parent comment.
2. Create a stream with several comments and replies. Like buttons should still work.
3. Click the "flag" button and reload the page. The like count on that comment should be unchanged.
4. Post a comment. The notification should not cover the last comment in the stream.

## Note
PR also allows the stream to be direclty accessed outside of an embed at `/embed/stream` for development purposes.

@coralproject/tech
